src/bivariate_normals.jl

```diff
@@ -126,14 +126,18 @@ immutable BvnComponent{NumType <: Number}
       c = 1 ./ (the_det^.5 * 2pi)
       major_sd = sqrt(maximum([ the_cov[1, 1], the_cov[2, 2] ]))
 
+      precision = Array(NumType, 2, 2)
+      precision[2, 2] = the_cov[1,1] / the_det
+      precision[1, 1] = the_cov[2,2] / the_det
+      precision[1, 2] = -the_cov[1, 2] / the_det
+      precision[2, 1] = -the_cov[2, 1] / the_det
+
       if calculate_siginv_deriv
         # Derivatives of Sigma^{-1} with respect to sigma.  These are the second
         # derivatives of log|Sigma| with respect to sigma.
         # dsiginv_dsig[a, b] is the derivative of sig^{-1}[a] / d sig[b]
         dsiginv_dsig = Array(ThisNumType, 3, 3)
 
-        precision = the_cov^-1
```

Comment:
I've changes the branch to use `StaticArrays` instead of `FixedSizeArrays` and tests are now passing (with a small patch to `StaticArrays`). The timing output from `benchmark_inter` goes from
```
24.936787 seconds (20.48 M allocations: 1.339 GB, 3.82% gc time)
10.575892 seconds (8.19 M allocations: 866.467 MB, 7.65% gc time)
```
on master to
```
23.577065 seconds (19.39 M allocations: 1.255 GB, 4.11% gc time)
10.268357 seconds (6.31 M allocations: 752.464 MB, 7.73% gc time)
```
on `anj/fixedsize`. So the effect is modest at this point.